data/sql/updates/pending_db_world/rev_1576659911412868500.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1576659911412868500');
+
+DELETE FROM `command` WHERE `name` IN ('bank', 'character check bank', 'character check bag', 'character check profession');
+
+INSERT INTO `command` VALUES
+('character check bank', 2, 'Syntax: .character check bank \r\n\r\nShow your bank inventory.'),
```

Comment:
does it also show the bank inventory of the selected player?

And what about allowing us to target a player by its name??? it could be useful

Also what happens if you use it in the console? maybe display a list of items or just say "you need to be logged ingame"?

Reply:
@IntelligentQuantum

Reply:
> does it also show the bank inventory of the selected player?
> 
> And what about allowing us to target a player by its name??? it could be useful
> 
> Also what happens if you use it in the console? maybe display a list of items or just say "you need to be logged ingame"?
> 
> otherwise, good work intelligentquantum!

Hey @BarbzYHOOL  :smile: 

1. Not just your bank
2. It's a good idea
3. The second case, if added, will also solve the console problem.

Reply:
Code it Quantum! Please :P @IntelligentQuantum